website/docs/components/toast/partials/guidelines/guidelines.md

```diff
@@ -51,8 +50,35 @@ Use color logically.
 - **Highlight** use interchangeably with `neutral` when more prominence is needed. Use it sparingly.
 - **Success** to indicate a successful action was completed.
 - **Warning** to indicate a successful action was completed but may have triggered a related issue. Provide guidance and actions if possible.
+
 - **Critical** to indicate error or critical issues resulting from a failed action.
 
+## Critical Toast vs Critical Alert
```

Comment:
[suggestion] It may make sense to nest this section under "Color" since it discusses a specific color variant.

Reply:
+1